app/src/main/res/values/strings.xml

```diff
@@ -17,15 +17,26 @@
 <resources>
     <string name="app_name">UAMP</string>
 
-    <string name="skip_back_10s">Skip back 10s</string>
-    <string name="skip_forward_10s">Skip forward 10s</string>
-    <string name="play">Play</string>
-    <string name="pause">Pause</string>
-
-    <string name="menu_music_queue">Queue</string>
     <string name="album_art_alt">Album art</string>
     <string name="duration_unknown">--:--</string>
     <string name="duration_format">%d:%02d</string>
 
-    <string name="cast_context_error">Failed to get Cast context. Try updating Google Play Services and restart the app.</string>
+    <string name="media_items_empty">Media Items Empty</string>
+    <string name="route_nowplaying">nowplaying</string>
```

Comment:
For these two routes, you can either just have them constants in Kotlin or you can have them here but add the `translatable="false"` attribute to each of them. For most apps, I'd recommend going the constant route because it means you don't need a Context reference in the code to use them, but this app only uses them where the Context is readily available, so either works fine. The translatable attribute is an indication that if someone comes in and creates a Spanish strings file, they don't need to translate the `translatable="false"` strings (since they're just used by code and not shown to users).